demo/index.html

```diff
@@ -29,24 +29,26 @@
   <link rel="import" href="../paper-item.html">
   <link rel="import" href="../paper-item-body.html">
   <link rel="import" href="../../paper-styles/color.html">
+  
+  <custom-style include="demo-pages-shared-styles">
```

Comment:
I don't think you need the `include="..."` here as long as it's on the `<style>` below.